oat-sa/extension-tao-delivery-rdf
# Ref #7842 - Test can not be published with empty item(s)

Ref #7842 - Test can not be published with empty item(s)
https://oat-sa.atlassian.net/browse/TAO-7842
Steps to reproduce: 
1. Log in as admin; 
2. Create some items, one of them should be empty; 
3. Create a test with created items including empty one; 
4. Publish created test;

Expected result: Error message should display when trying to publish test. 
Actual result: Test published and can be runned by TT

The test items are checked for emptiness. If there are no elements, the error is generated and sent to the client.
